testing/test_capture.py

```diff
@@ -1243,7 +1269,7 @@ def test_capsys_results_accessible_by_attribute(capsys):
 def test_fdcapture_tmpfile_remains_the_same(tmpfile, use):
     if not use:
         tmpfile = True
```

Comment:
this test looks strange, whats the value of tmpfile?

Reply:
Right, I'm not sure what this was supposed to do either (it's the same in master -- the two parametrization are actually testing the same thing). I added a commit which removes the parametrization.